packages/cursorless-everywhere-talon-core/src/polyfill.ts

```diff
@@ -0,0 +1,20 @@
+const global = globalThis as any;
+
+if (global.process == null) {
+  global.process = {
+    env: {},
+  };
+}
+
+if (typeof print !== "undefined") {
+  global.console = {
+    log: print,
+    error: print,
+    warn: print,
+    debug: print,
+  };
+}
+
+global.setTimeout = (callback: () => void, _delay: number) => {
+  callback();
+};
```

Comment:
As discussed, I'm not in love with silently ignoring the delay. Do we actually need this polyfill?

Reply:
Yes. This gets called by our hat map debouncer.

Reply:
But if we pass null hats we never construct hat map right?

Reply:
filed https://github.com/cursorless-dev/cursorless/issues/2596; let's add a FIXME as well pointing to it